src/app/api/participant.py

```diff
@@ -0,0 +1,39 @@
+from app.RatingCalculator import RatingCalculator
+from app.api.base import BaseAuthResource
+from app.participant.models import ParticipantModel, create_participant, delete_participant, update_participant
+from flask.ext.restful import marshal, fields
+
+__author__ = 'Alex'
+
+participant_template = {
+    'league_id': fields.String,
+    'participant_id': fields.String,
+    'name': fields.String,
+    'rating': fields.String
+}
+
+
+class ParticipantListAPI(BaseAuthResource):
+    def get(self, league_id):
+        participants = ParticipantModel.query(league_id=league_id).fetch()
+        return {'data': [marshal(1, participant_template) for 1 in participants]}
+
+    def post(self):
```

Comment:
Since the route you assign this class to in main.py is ``` '/participant/<string:league_id>' ```, you're ALWAYS going to get league_id as an argument to this function. Use that instead of trying to look for it in the args below.